hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/HoodieFileGroupReaderBasedFileFormat.scala

```diff
@@ -87,6 +88,14 @@ class HoodieFileGroupReaderBasedFileFormat(tablePath: String,
 
   private lazy val avroTableSchema = new Schema.Parser().parse(tableSchema.avroSchemaStr)
 
+  private lazy val tableSchemaAsMessageType: org.apache.hudi.common.util.Option[org.apache.parquet.schema.MessageType] = {
+    org.apache.hudi.common.util.Option.ofNullable(
+      ParquetTableSchemaResolver.convertAvroSchemaToParquet(avroTableSchema, new Configuration())
+    )
+  }
+
+  private lazy val supportBatchTimestampRepair = HoodieSparkUtils.gteqSpark3_5 || !AvroSchemaRepair.hasTimestampMillisField(avroTableSchema)
```

Comment:
So there is no way to enable vectorized reader support on Spark 3.3 and 3.4 if there is no timestamp-millis field in the table schema?

Reply:
?? if there's a !

Reply:
Ah, got it.  Make sense now.